### PR TITLE
feat: add `require_comment` option to environment settings 

### PR DESCRIPTION
**Pull request by cre8ivejp**
Part of https://github.com/bucketeer-io/bucketeer/issues/866

### CODE REVIEW

File: pkg/locale/localizedata/en.yaml [hunk 1]
```diff
@@ -115,6 +115,7 @@ ConditionDeleted: "{{ .Field_1 }} condition has been deleted"
 Changed: "{{ .Field_1 }} has been changed"
 Updated: "{{ .Field_1 }} has been updated"
 DescriptionUpdated: "{{ .Field_1 }} description has been updated"
+EnvironmentRequireCommentUpdated: "Environment settings have been updated"
```
Comment (kentakozuka) [inline pkg/locale/localizedata/en.yaml:118]: How about adding `EnvironmentSettings` to [here](https://github.com/bucketeer-io/bucketeer/blob/ee1c0f51226730b0c8aa54067c37c339c9c2a95d/pkg/locale/localizer.go#L108) and using `locale.UpdatedTemplate`?

**Review by kentakozuka — COMMENTED** [review]
Nice! left a comment. PTAL 👀

File: pkg/locale/localizedata/en.yaml (hunk 1 above)
Comment (cre8ivejp) [reply pkg/locale/localizedata/en.yaml]: @kentakozuka, done.

**Review by Ubisoft-potato — COMMENTED** [review]
LGTM! 👍

**Review by kentakozuka — APPROVED** [review]
LGTM 👍

**Pull request merged**